specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,18 +28,27 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2021-03
+tag: package-preview-2021-05
 ```
 
 
+### Tag: package-preview-2021-05
+
+These settings apply only when `--tag=package-preview-2021-05` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2021-05'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensions.json
```

Comment:
Preview file needs to contain both input-files:
- Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensions.json
- Microsoft.KubernetesConfiguration/stable/2021-03-01/kuberentesconfguration.json